doc/manual/src/glossary.md

```diff
@@ -150,7 +152,23 @@
     store. It can contain regular files, directories and symbolic
     links.  NARs are generated and unpacked using `nix-store --dump`
     and `nix-store --restore`.
+
   - [`∅`]{#gloss-emtpy-set}\
     The empty set symbol. In the context of profile history, this denotes a package is not present in a particular version of the profile.
+
   - [`ε`]{#gloss-epsilon}\
     The epsilon symbol. In the context of a package, this means the version is empty. More precisely, the derivation does not have a version attribute.
+
+  - [string interpolation]{#gloss-string-interpolation}\
```

Comment:
I don't think this should be in the glossary. It should be a section in the language chapter. The glossary should just have brief definitions that link elsewhere for full details.

Reply:
Done